flake.nix

```diff
@@ -1,18 +1,23 @@
 {
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.11";
     flake-utils.url = "github:numtide/flake-utils";
     fedimint = {
+      # temporarily switched to https://github.com/fedimint/fedimint/pull/4349
       url =
-        "github:fedimint/fedimint?rev=9ec5418ee002134188d4584cc645fe1b5ffba4f1";
+        "github:dpc/fedimint?rev=678d456a1e242fb952b92514f0e6a83083a9704d";
```

Comment:
@dpc taking over the UI repo :P

Reply:
I wonder if we should have a CI check that our flake inputs always use a tag to specify the dependency? When it's just a git hash, you can't tell by inspection what version is being used ...